scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -153,7 +163,7 @@ workflow GvsCreateFilterSet {
 
   # support a custom truth file
   # Specifying default resources here to make condition below clearer to read
-  String default_hg38_resources = "--resource:hapmap,training=true,calibration=true ${GetReference.reference.hapmap_resource_vcf} --resource:omni,training=true,calibration=true ${GetReference.reference.omni_resource_vcf} --resource:1000G,training=true,calibration=false ${GetReference.reference.one_thousand_genomes_resource_vcf} --resource:mills,training=true,calibration=true ${GetReference.reference.mills_resource_vcf} --resource:axiom,training=true,calibration=false ${GetReference.reference.axiomPoly_resource_vcf}"
+  String default_hg38_resources = "--resource:hapmap,training=true,calibration=true ${GetResources.resources.hapmap_resource_vcf} --resource:omni,training=true,calibration=true ${GetResources.resources.omni_resource_vcf} --resource:1000G,training=true,calibration=false ${GetResources.resources.one_thousand_genomes_resource_vcf} --resource:mills,training=true,calibration=true ${GetResources.resources.mills_resource_vcf} --resource:axiom,training=true,calibration=false ${GetResources.resources.axiomPoly_resource_vcf}"
```

Comment:
I get this is preexisting, but do you know why the `${foo}` syntax is used here?

Reply:
Probably my work from a previous PR. I'll update it.